server/pulp/server/managers/auth/permission/cud.py

```diff
@@ -129,7 +116,7 @@ def grant(resource, login, operations):
 
         user = User.get_collection().find_one({'login': login})
         if user is None:
-            raise MissingResource(user=login)
+            raise InvalidValue(['login'])
```

Comment:
Can the docstrings for these functions that raise `InvalidValue` have a `:raises InvalidValue:` added to them along with a reasonable description of when it gets raised? You didn't create this problem, but I do notice it in the PR. Similarly throughout the PR where functions raise `InvalidValue`.